index.js

```diff
@@ -78,15 +78,32 @@ function generateAuthToken() {
 
   const imsOrgId = core.getInput('imsOrgId')
 
+  let parsedScopes
+  try {
+    parsedScopes = JSON.parse(scopes)
+  } catch (err) {
+    throw new Error('SCOPES environment variable must be a valid array of strings with double quotes (e.g. ["ent_adobeio_sdk"]) to use the auth command')
+  }
+
+  if (!parsedScopes) {
+    throw new Error('SCOPES environment variable must be defined to use the auth command')
+  }
+
+  if (!Array.isArray(parsedScopes)) {
+    throw new Error('SCOPES environment variable must be an array of strings with double quotes (e.g. ["test_scope"]) to use the auth command')
+  }
+
+  if (!parsedScopes.includes('ent_adobeio_sdk')) {
+    throw new Error('SCOPES environment variable must include the "ent_adobeio_sdk" scope (e.g. ["ent_adobeio_sdk"]) to use the auth command')
+  }
+
   const imsConfig = {
     client_id : clientId,
     client_secret: clientSecret,
     technical_account_id: techAccId,
     ims_org_id: imsOrgId,
     private_key: key.toString(),
-    meta_scopes: [
```

Comment:
All CI/CD action that work today will be using comma separated string format for scopes.
Although its different than the documented format, but this might be working for most of the users.
On second thought I feel we should just update the readme to say comma separated values are supported and keep the code as is. 
The current change to force array based format will break CI/CD for all working GH workflows.

Reply:
Even if we decide to change the format to a more standard/canonical format - we can always release a major version and that will not break existing users.

Reply:
Good point. Still might be good for us to verify which formats work so we can update the README

Reply:
To sidestep all the formatting confusion, we are going to try hard-coding the required scopes and requesting them for users

Since the only error that can occur when the hard-coded scopes are being requested is that the credential doesn't have access to them, we can include a call to action in the error message to add the required APIs to the credential and then retry